### PR TITLE
chore: added new resources to ui-libraries

### LINKED ISSUE #2035: [OTHER] add Bootstrap , semantic UI and SeamlessUI to UI libraries
Location : Frontend/UI libraries

add Bootstrap , Semantic UI and Seamless UI links .

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by hritikbhattacharya**
Closes #2035

## Changes proposed

Added the Links for bootstrap , SemanticUI , seamlessUI to the UI libraries.

### CODE REVIEW

File: components/TopBar/CategoryDescriptions.ts [hunk 1]
```diff
@@ -94,6 +94,8 @@ const categoryDescriptions: CategoryDescriptions = {
     'Anyone can contribute to the projects are called OpenSource Projects. There are many open source projects available for developers. Some of the most popular ones include Linux, Git, and Apache1. Other open source projects include Kubernetes, TensorFlow, and React Native',
   tools:
     'There are various open source tools available for developers. Some of the most popular ones include Hadoop, VLC Media Player, and GIMP1. Other open source tools include TensorFlow, Eclipse Che, and OpenShift Do.',
+  'os tutorials':
+    'OS tutorials helps you to learn-by-watching. It contains the list most helping tutorials on open-source.',
```
Comment (Anmol-Baranwal) [inline components/TopBar/CategoryDescriptions.ts:98]: Why have you added this?

**Review by Anmol-Baranwal — COMMENTED** [review]
Let me know the reason, because I cannot find a appropriate subcategory for that description.

File: components/TopBar/CategoryDescriptions.ts (hunk 1 above)
Comment (aftabrehan) [reply components/TopBar/CategoryDescriptions.ts:98]: @hritikbhattacharya, This simply isn't required. We have renamed the `os tutorials` as `tutorials`, and its description has already been added [here](https://github.com/rupali-codes/LinksHub/pull/2180/files#diff-6255e5230971ba71d0f0818df0f73b28825755e29ede66a7336941cc5a5e4c80R99-R100).

Kindly, please revert this change.

```suggestion
```

Comment (hritikbhattacharya) [reply components/TopBar/CategoryDescriptions.ts]: removed

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**